sim/midas/src/main/cc/bridges/reset_pulse.cc

```diff
@@ -0,0 +1,39 @@
+#ifdef RESETPULSEBRIDGEMODULE_struct_guard
```

Comment:
What is this `RESETPULSEBRIDGEMODULE_struct_guard` macro and how does it get defined?  I'm not seeing that in this pr, is it just letting you have this code in your branch but not use it in the driver right now?  Is this supposed to be taken out before merging?

Reply:
This is a necessary (evil) component of defining a bridge since one of the constructor arguments is defined for the first time in the generated header. If no bridges are instantiated, the struct is not defined, and this code cannot compile...